src/popup/popup.html

```diff
@@ -132,7 +132,17 @@ <h2>The Akita Icon</h2>
 					data="../../assets/demo_unmon.svg"
 					style="height: 100px;">
 				</object>
-				<p>Thank you for reading.</p>
+			</div>
+			<div class="carousel-slide">
+				<h2>About Your Data</h2>
+				<p>Akita collects data on</p>
+				<ul style="line-height: normal;">
+					<li>your time spent and number of visits on all websites,</li>
+					<li>the Payment Pointers present on the websites you visit,</li>
```

Comment:
Do we want to link to the payment pointer site here too?

Reply:
Thought it could be a bit technical, but I suppose there's no harm. Will do.